CSS fixes for Swage

Inverle opened the pull request:
- Fixes for Swage
	- Buttons not showing correctly #7465
	- Dropdown placement (some dropdowns couldn't fit in viewport)
	- Icons not appearing in some places due to color i.e. label management and subscription management page
	- Made `.form-group` and button inside search dropdown hover colors slightly darker, for better visibility
	- Nav menu style improvements on mobile
		- Smaller if not on main page
		- Align settings icon on configuration pages to the right
		- Support nav menu for feed statistics page
	- Text alignment in search dropdown
	- Ensure `input, select, textarea` don't overflow the page
	- Login link placement in anonymous view
	- Other fixes for
		- Add via bookmarklet page
		- About page
		- Shared HTML query page
		- Register page
	- Text visibility for debug log
- Provide classes `layout.phtml` within `<html>`:
	- `logged_in` if applicable
	- `controller_$NAME` if applicable
	- `file_$NAME` if applicable
- Some other fixes

## Moved to #7616
### the PR is still dependent on these changes, and isn't complete without them
- ~~Named some elements with IDs in `nav_menu.phtml` to make them easier selectable in Swage~~
- ~~Fixes for frss.css~~
	- ~~It wasn't possible to scroll fully to the bottom in aside menu and configure menu on mobile~~
	- ~~Align text on expandable sections to the left on mobile~~
	- ~~Show `not-allowed` cursor on disabled buttons instead of a `pointer`~~

Tested on Chrome, Firefox and GNOME Web

Inverle committed:
CSS fixes for Swage + some in frss.css

Inverle commented:
QuickCollapse button for example is still broken though, but only on mobile

Alkarex commented:
For Swage, the changes need to be done in https://github.com/FreshRSS/FreshRSS/blob/edge/p/themes/Swage/swage.scss
and then run:

```sh
sass swage.scss:swage.css
```

As a side note, I would like to get rid of SCSS at some point to use only standard CSS, using [CSS nesting](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting). It should be possible to make an automatic conversion from our current SCSS files used by some of our themes. Help welcome.

Inverle committed:
Use SASS and make dropdowns full width on mobile

Inverle commented:
> [make dropdowns full width on mobile](https://github.com/FreshRSS/FreshRSS/pull/7608/commits/37fa68129cc0f8ed2b289ba3497191deead6e5b7)

turns out I didn't fix them correctly

Inverle committed:
Merge branch 'edge' into swage-css-fixes

math-GH commented on app/layout/nav_menu.phtml:
```diff
@@ -4,13 +4,13 @@
 
 	<?php if (FreshRSS_Auth::hasAccess()) { ?>
-	<div id="nav_menu_actions" class="group">
+	<div class="group" id="nav_menu_actions">
```
I prefer to have the `id` first before `class` because `id` is unique and stronger compared to `class`

Inverle replied:
done

math-GH commented on p/themes/Swage/swage.css:
```diff
@@ -1,1182 +1,1274 @@
 :root {
-	--color-text-light: #fcfcfc;
-	--color-text-light-darker: #969696;
-	--color-text-dark: #181621;
-	--color-text-nav: rgb(0, 71.6947368421, 139);
-	--color-text-aside: #22303d;
```
Please check the text indentation and respect the given.

Inverle replied:
I'm not sure why that happened

math-GH commented on p/themes/Swage/swage.css:
```diff
@@ -1,1182 +1,1274 @@
+  --color-text-good: #5eaabf;
+  --color-text-bad: #b0425b;
+  --color-text-bad-lighter: rgb(195.8181818182, 97.1818181818, 119.5991735537);
+  --color-text-shadow-light: #f5f3f3;
+  --color-box-shadow-light: rgb(221.8181818182, 215.1818181818, 215.1818181818);
+  --color-border-light: #f5f3f3;
+  --color-border-light-darker: rgb(221.8181818182, 215.1818181818, 215.1818181818);
+  --color-border-grey: #e3e3e3;
```
Could you please give examples why you change so many colors? 
Would it be possible to have cleaner numbers instead of having 10  positions after decimal point?

Alkarex replied:
I think that is due to the current SASS default conversion when running `sass swage.scss:swage.css`. It may require an option to the command line. Suggestion welcome

Alkarex replied:
By the way, as written above:

> As a side note, I would like to get rid of SCSS at some point to use only standard CSS, using [CSS nesting](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting). It should be possible to make an automatic conversion from our current SCSS files used by some of our themes. Help welcome.

which would also solve such issues

Inverle replied:
Yes, it is due to SASS, I changed one color variable inside the .scss file, and that changed more colors.
It may be enough to just change the color `--color-background-light` in `.form-group:hover`, where I specifically need it:

https://github.com/FreshRSS/FreshRSS/blob/37fa68129cc0f8ed2b289ba3497191deead6e5b7/p/themes/Swage/swage.scss#L235-L243

I think it's fine as is though.

math-GH replied:
yes ,I agree to get rid of SCSS (it is a task outside this PR). But before we should check the compatibility with older devices and browser. CSS nesting is quite new

Alkarex replied:
Yes for another PR.
CSS nesting is "Baseline 2023", so I think it is good enough to migrate some of our non-default themes. In case of problem, a user can fall back to another theme

Alkarex replied:
* https://github.com/FreshRSS/FreshRSS/issues/7611

math-GH commented on p/themes/Swage/swage.css:
```diff
@@ -1,1182 +1,1274 @@
+
+#slider-content #feed_update fieldset .form-group:last-child {
+  margin-bottom: 0.4rem;
+}
```
Please use `rem` instead of `px` (`4px` = `0.25rem`

math-GH commented on p/themes/Swage/swage.css:
```diff
@@ -1,1182 +1,1274 @@
+  content: url(../Swage/icons/error.svg);
+}
+
+main.post > div.form-group.form-actions > div.group-controls > ul > li > a {
+  margin-left: 1rem;
 }
```
It looks complicated. Why is it needed?

Inverle replied:
Used on the import/export page for mobile

After:

![image](https://github.com/user-attachments/assets/eb224aaf-362b-419d-9f1d-bf1a4dffa755)

Before:

![image](https://github.com/user-attachments/assets/d93c3723-b8a3-46bc-b357-83fd9601d7b3)

math-GH replied:
good catch. Let's find a better solution. Please check the import button on the same page below "Dynamic OPML" section. The `import` button has a `margin` too. 

This is the CSS line for the button:
```
.form-group.form-actions .btn {
  margin: 0 10px;
}
```

would 
```
.form-group.form-actions .btn,
.form-group.form-actions .a {
...
```
work too?

Inverle replied:
yes, done

Inverle commented:
I broke the actualize button from appearing after moving changes from css to scss, so I'll have to fix that too.

Inverle committed:
Fixes and suggested changes

Inverle commented:
todo:
- [x] ~~sign in button is incorrectly positioned on anonymous view~~
- [x] ~~the search is hidden for some reason (on anonymous view)~~
- [x] ~~about page is broken~~
- [x] ~~add feed via bookmarklet page is broken~~

Inverle committed:
Further fixes

Inverle committed:
Further fixes v2

Inverle committed:
make fix-all

math-GH commented:
to be honest: this PR blewed up to much more than just CSS fixes for Swage. 

I would suggest to split the PR and keep the focus here on Swage

Inverle commented:
> to be honest: this PR blewed up to much more than just CSS fixes for Swage.

Yes.
- Fixes for Swage
- Fixes for frss.css that also affect Swage
- Added necessary ids and class names in layout.phtml and nav_menu.phtml to use them in Swage
	- also moved an if block in nav_menu.phtml

> I would suggest to split the PR and keep the focus here on Swage

Which parts?

math-GH commented:
This parts could be a separate PR because it will affect also the other themes or the other themes could be benefit from it too:

>     * Provide `logged_in` class within `<html>` if applicable, and a controller class
> 
>     * Fixes for frss.css
>       
>       * It wasn't possible to scroll fully to the bottom in aside menu and configure menu on mobile
>       * Align text on expandable sections to the left on mobile
>       * Show `not-allowed` cursor on disabled buttons instead of a `pointer`
> 
>     * Some other fixes
> 
>     * Made more navigation buttons accessible within anonymous view
>

Inverle commented:
> * Provide `logged_in` class within `<html>` if applicable, and a controller class

It doesn't look like any other themes aside from Swage can benefit from those changes

> * Fixes for frss.css

I guess this does affect all themes, not just Swage

> * Some other fixes

Those are just for Swage (adding borders, margins and paddings in some places)

> * Made more navigation buttons accessible within anonymous view

Yes, this part would make sense, but I will also make the user queries dropdown visible (I haven't made it visible yet, I'll have to also hide the `bookmark current query` button inside the user queries dropdown)

Before:

![image](https://github.com/user-attachments/assets/a9c7d6be-c387-495a-bfb5-0bb8182d1bb7)


After (still have to make user queries visible):

![image](https://github.com/user-attachments/assets/344321ca-b6de-4e9b-a431-a0fe79b3fd98)

Inverle commented:
Done #7616

Inverle commented:
todo:
- [x] ~~fix shared html query page~~

Inverle committed:
Further fixes v3

Inverle committed:
PHPStan fix

Inverle committed:
PHPStan fix v2

math-GH commented on app/layout/layout.phtml:
```diff
@@ -2,19 +2,27 @@
 	/** @var FreshRSS_View $this */
 	FreshRSS::preLayout();
-	$class = '';
+	$class = [];
+	if (Minz_Request::controllerName() !== '') {
+		$class[] = 'controller_' . Minz_Request::controllerName();
+	} else {
+		$class[] = 'file_' . pathinfo(is_string($_SERVER['SCRIPT_NAME'] ?? null) ? $_SERVER['SCRIPT_NAME'] : '', PATHINFO_FILENAME);
+	}
```
for what is it needed?
It looks very complicated. How is it connected to the CSS fixes for Swage?

Inverle replied:
If you look in swage.scss where I'm using `:root.file_*` then you'll see that I'm using it specifically for `:root.file_query` (which is the HTML user query sharing page `/api/query.php?user=...&t...=&f=html`)

After:

![image](https://github.com/user-attachments/assets/c127a272-4ef1-4b33-81a9-4e2a091fcfa2)

Before:

![image](https://github.com/user-attachments/assets/750fa94f-7602-4187-ba81-fcf9607ef259)

And then I also need the controller class e.g. `:root.controller_feed`, because there are both `/i/?c=feed&a=subscription` and `/i/?c=feed&a=add` (used for bookmarklet add) endpoints

`/i/?c=feed&a=add` after:

![image](https://github.com/user-attachments/assets/0f1a5fec-b14e-4d23-9649-e778f96ced26)

`/i/?c=feed&a=add` before:

![image](https://github.com/user-attachments/assets/0cbe333a-6fd8-4952-a4fa-696fb74136b7)


edit: those classes exist because selecting just the action name from the `body` class name is not sufficient

Inverle replied:
I just noticed that the search field is missing from the API query page, so this may need more work.

Inverle replied:
Needs a separate PR to solve, as this is also broken for other themes. e.g. Origine shows search bar only when not on mobile

(unless I just match the behavior of Origine for now)

Inverle replied:
Bookmarklet page header looks like this now

<img width="1864" height="245" alt="image" src="https://github.com/user-attachments/assets/f1edb940-ab17-4067-ac53-9417a6f9f863" />

math-GH commented on app/layout/nav_menu.phtml:
```diff
@@ -4,12 +4,11 @@
 		</div>
 	<?php } ?>
 
-	<?php if (FreshRSS_Auth::hasAccess()) { ?>
```
is not it part of https://github.com/FreshRSS/FreshRSS/pull/7616?

Inverle replied:
Yes, it's no longer supposed to be in this PR

math-GH commented on app/layout/nav_menu.phtml:
```diff
@@ -65,7 +64,8 @@
 		</div>
 
-		<div class="dropdown">
+		<?php if (FreshRSS_Auth::hasAccess()) { ?>
```
is not it part of https://github.com/FreshRSS/FreshRSS/pull/7616?

Inverle replied:
Yes, it's no longer supposed to be in this PR

math-GH commented on app/layout/nav_menu.phtml:
```diff
@@ -181,11 +181,11 @@
 			<a class="dropdown-close" href="#close">❌</a>
 		</div>
 		</form>
+		<?php } ?>
```
is not it part of https://github.com/FreshRSS/FreshRSS/pull/7616?

Inverle replied:
Yes, it's no longer supposed to be in this PR

math-GH commented on p/themes/base-theme/frss.css:
```diff
@@ -2474,6 +2478,20 @@ html.slider-active {
 	}
 
+
+	body:not(.normal):not(.reader):not(.global) .aside {
+		overflow-y: scroll;
+	}
+
```
Which problem does it solve?

Inverle replied:
> It wasn't possible to scroll fully to the bottom in aside menu and configure menu on mobile

It's no longer supposed to be in this PR.

math-GH commented on p/themes/base-theme/frss.css:
```diff
@@ -2474,6 +2478,20 @@ html.slider-active {
+		margin-bottom: 6rem;
+	}
+
+	.form-advanced-title {
+		text-align: left;
+	}
```
where is it used?

Inverle replied:
> Align text on expandable sections to the left on mobile

It's no longer supposed to be in this PR.

math-GH replied:
could you please check your commits and pushs? Currently this code change is still part of this PR

Inverle replied:
@math-GH I reverted the changes from this PR that are now in #7616. But #7616 needs to be merged first, because lack of necessary changes in nav_menu.phtml now broke some parts of this PR.

(I didn't exactly foresee that this would happen)

math-GH commented on p/themes/base-theme/frss.css:
```diff
@@ -2474,6 +2478,20 @@ html.slider-active {
+		overflow-y: scroll;
+	}
+
+	nav.configure > .dropdown li.item.dropdown-section:last-child,
+	nav#aside_feed > ul li.item.nav-section:last-child {
+		margin-bottom: 6rem;
```
Which problem does it solve?
As far as I can oversee it could risk a scrolling where it is not needed.

Inverle replied:
> It wasn't possible to scroll fully to the bottom in aside menu and configure menu on mobile

It's no longer supposed to be in this PR.

> As far as I can oversee it could risk a scrolling where it is not needed.

It's in the media query for mobile, and for specific elements, so I don't think there are any risks.

Inverle committed:
Revert moved changes

Inverle committed:
Fix search bar for API query page and input focus styles

Inverle committed:
Register page fixes

math-GH commented:
Please check the reader view:

before:
![grafik](https://github.com/user-attachments/assets/3a1d89bd-4da2-4a66-ae11-2a19745cc722)

this PR:
the buttons are not placed well now
![grafik](https://github.com/user-attachments/assets/5e7d6547-5175-4309-9689-025ae5b9fc43)

Inverle commented:
> Please check the reader view:
> 
> before: ![grafik](https://private-user-images.githubusercontent.com/1645099/449839147-3a1d89bd-4da2-4a66-ae11-2a19745cc722.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDg4MDk5MjksIm5iZiI6MTc0ODgwOTYyOSwicGF0aCI6Ii8xNjQ1MDk5LzQ0OTgzOTE0Ny0zYTFkODliZC00ZGEyLTRhNjYtYWUxMS0yYTE5NzQ1Y2M3MjIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDYwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA2MDFUMjAyNzA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDRhMDk4OGMxYmYyZTRiOTZkZmE2Njg0MDFiZWQ3YWFjMjRjODhiMzc0MjdiNTEzMjQ4N2ExMGM1ZGM1NmZhOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.xo0Qb5Z9NGnQ8FXUYyVr_K8i9bOwvsWKGxNDyDRlamc)
> 
> this PR: the buttons are not placed well now ![grafik](https://private-user-images.githubusercontent.com/1645099/449839184-5e7d6547-5175-4309-9689-025ae5b9fc43.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDg4MDk5MjksIm5iZiI6MTc0ODgwOTYyOSwicGF0aCI6Ii8xNjQ1MDk5LzQ0OTgzOTE4NC01ZTdkNjU0Ny01MTc1LTQzMDktOTY4OS0wMjVhZTViOWZjNDMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDYwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA2MDFUMjAyNzA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjMyNTQyNWFiYTJmNGNhYzU5ZTlkNGJkNzMyNWI2OThiYjkwNjc3ZWE0ZjQ0YzQzZTFhNzFkM2E4MTczNTI3YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Sfie6ba--X_tAWMf0dUe24zAz93J79OaSpkPz95Jlok)

Make sure to also apply the changes from #7616

Inverle committed:
Fix text visibility for debug log

math-GH commented:
> > Please check the reader view:
> > before: ![grafik](https://private-user-images.githubusercontent.com/1645099/449839147-3a1d89bd-4da2-4a66-ae11-2a19745cc722.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDg4MDk5MjksIm5iZiI6MTc0ODgwOTYyOSwicGF0aCI6Ii8xNjQ1MDk5LzQ0OTgzOTE0Ny0zYTFkODliZC00ZGEyLTRhNjYtYWUxMS0yYTE5NzQ1Y2M3MjIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDYwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA2MDFUMjAyNzA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDRhMDk4OGMxYmYyZTRiOTZkZmE2Njg0MDFiZWQ3YWFjMjRjODhiMzc0MjdiNTEzMjQ4N2ExMGM1ZGM1NmZhOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.xo0Qb5Z9NGnQ8FXUYyVr_K8i9bOwvsWKGxNDyDRlamc)
> > this PR: the buttons are not placed well now ![grafik](https://private-user-images.githubusercontent.com/1645099/449839184-5e7d6547-5175-4309-9689-025ae5b9fc43.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDg4MDk5MjksIm5iZiI6MTc0ODgwOTYyOSwicGF0aCI6Ii8xNjQ1MDk5LzQ0OTgzOTE4NC01ZTdkNjU0Ny01MTc1LTQzMDktOTY4OS0wMjVhZTViOWZjNDMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDYwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA2MDFUMjAyNzA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjMyNTQyNWFiYTJmNGNhYzU5ZTlkNGJkNzMyNWI2OThiYjkwNjc3ZWE0ZjQ0YzQzZTFhNzFkM2E4MTczNTI3YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Sfie6ba--X_tAWMf0dUe24zAz93J79OaSpkPz95Jlok)
> 
> Make sure to also apply the changes from #7616

I prefer independent PRs. In this case: the other PR (7616) is still in discussion.

Inverle commented:
> I prefer independent PRs. In this case: the other PR (7616) is still in discussion.

I was saying that this PR can't be tested without the changes from the other PR, which is why you were seeing a broken nav menu.

Alkarex committed:
Merge branch 'edge' into swage-css-fixes

Inverle committed:
Merge branch 'edge' into swage-css-fixes

Inverle committed:
Merge branch 'edge' into swage-css-fixes

Inverle committed:
Remove some hacky CSS

Still needs improvement though

Inverle commented:
Known bugs:

* (in settings after scrolling down)

<img width="211" height="240" alt="image" src="https://github.com/user-attachments/assets/a3c3308a-e18d-4f5d-925e-417656561aa4" />

* Articles don't load automatically when scrolling down in stream for some reason

Alkarex commented on p/themes/Swage/swage.scss:
```diff
@@ -382,6 +457,12 @@ form {
+	.item > a,
+	.item > .as-link,
+	.item > span {
+		width: 96% !important;
```
If possible, it would be better to avoid the `!important` like here and use a more precise qualification instead.
But may wait for later

Inverle replied:
Still WIP until I fix the issues mentioned in https://github.com/FreshRSS/FreshRSS/pull/7608#issuecomment-3156668298

(and maybe do some more refactoring of the CSS)

Alkarex replied:
Ok, I have just marked the PR as draft then

Inverle commented:
> Known bugs:
> 
>     * (in settings after scrolling down)
> 
> 
> <img alt="image" width="211" height="240" src="https://private-user-images.githubusercontent.com/205687958/474722290-a3c3308a-e18d-4f5d-925e-417656561aa4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQ1MTEwODcsIm5iZiI6MTc1NDUxMDc4NywicGF0aCI6Ii8yMDU2ODc5NTgvNDc0NzIyMjkwLWEzYzMzMDhhLWUxOGQtNGY1ZC05MjVlLTQxNzY1NjU2MWFhNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwNlQyMDA2MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xM2ExMTFiMzE3MTVkZTAwN2NmMjZlOTFhNTA2MmZiZGQ1YjBiNWU3ZjYyNjY0Mjc2NWQwYTg5OTVkMjI5N2I2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.bGeEukTULpDcYYCquDjMJGCNMEFDSPUZpmM8LPOMMrA">
> 
>     * Articles don't load automatically when scrolling down in stream for some reason

both of the issues were caused by `overflow-x: hidden` in `<html>` added in previous commit

Inverle committed:
More fixes

Inverle commented:
Anonymous view when more buttons were shown (mobile view)
<img width="682" height="64" alt="image" src="https://github.com/user-attachments/assets/0e9b33ad-140d-4f27-b242-a1ea82e7c17b" />
Now:
<img width="681" height="66" alt="image" src="https://github.com/user-attachments/assets/efeddb26-2f04-4890-bb56-2b9b64878edf" />
A little empty

Inverle committed:
Fix login button in anonymous view

Inverle committed:
stylelint fix

Inverle committed:
Merge branch 'edge' into swage-css-fixes

Inverle commented:
I consider it ready but testing welcome

Alkarex commented:
Yes, I would like to take time to look a bit closer to the changes in `layout.phtml` but would first like to release FreshRSS 1.27.0.
This PR can land in a minor `1.27.1` or whichever is the next release.

Inverle committed:
Make header UI consistent on bookmarklet page

Inverle commented:
> Yes, I would like to take time to look a bit closer to the changes in layout.phtml

I did just find one issue, fixing it right now

Inverle committed:
Remove whitespace from classes

Inverle commented:
Resolves the same `x dropdown-menu` issue as in #7722 (tested with darkMode field from user config)

Inverle committed:
Simplify giant selector for header style

Inverle committed:
Remove leftover moved CSS

Inverle commented:
What's the specific concern with layout.phtml?
@Alkarex

math-GH committed:
Merge branch 'edge' into swage-css-fixes

math-GH commented:
It looks now ok for me.

Alkarex committed:
Minor whitespace

Alkarex committed:
Merge branch 'edge' into pr/Inverle/7608

Alkarex commented:
The SCSS annoyances will vanish when we get to implement:
* https://github.com/FreshRSS/FreshRSS/issues/7611
